_docs-sources/iac/getting-started/deploying-a-module.md

```diff
@@ -1,27 +1,267 @@
+Modules allow you to define an interface to create one or many resources in the cloud or on-premise. Similar to how in object oriented programming you can define a class that may have different attribute values across many instances.
 
-## Create a module
+Modules help create your Terraform code DRY (don't repeat yourself), and speed up development time when creating new resources.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Hac habitasse platea dictumst quisque sagittis purus sit. Eu scelerisque felis imperdiet proin fermentum leo vel orci porta. Ornare arcu dui vivamus arcu felis bibendum ut tristique et. Arcu ac tortor dignissim convallis aenean et. Augue interdum velit euismod in pellentesque massa placerat duis. Turpis tincidunt id aliquet risus feugiat in ante metus dictum. Vel pharetra vel turpis nunc eget. Ac turpis egestas sed tempus urna et. Scelerisque in dictum non consectetur. Nibh sit amet commodo nulla facilisi nullam vehicula ipsum a. Ornare aenean euismod elementum nisi quis eleifend quam adipiscing vitae. At varius vel pharetra vel turpis. Mattis ullamcorper velit sed ullamcorper morbi tincidunt ornare massa.
+This tutorial will teach you how to develop a Terraform module that deploys an AWS Lambda. We will create the required file structure, define an AWS Lambda and AWS IAM role as code, then plan and apply the resource in an AWS account. Then, we’ll verify the deployment by invoking the Lambda using the AWS CLI.
 
-## Run terraform plan
+## Prerequisites
+- An AWS account with permissions to create the necessary resources.
+- An [AWS Identity and Access Management](https://aws.amazon.com/iam/) (IAM) user or role with permissions to create IAM roles and AWS Lambdas
+- [AWS Command Line Interface](https://aws.amazon.com/cli/) (AWS CLI) installed on your local machine
+- [Terraform](https://www.terraform.io) installed on your local machine
 
-Lobortis feugiat vivamus at augue eget. Est placerat in egestas erat imperdiet sed euismod nisi porta. Volutpat maecenas volutpat blandit aliquam etiam erat. Vulputate mi sit amet mauris commodo quis imperdiet massa tincidunt. Nec tincidunt praesent semper feugiat. Accumsan sit amet nulla facilisi morbi. Euismod in pellentesque massa placerat duis ultricies. Amet aliquam id diam maecenas ultricies mi eget. Et tortor consequat id porta nibh venenatis cras sed felis. Lacus vel facilisis volutpat est velit egestas dui id.
+## Create the module
 
-## Run terraform apply
+In this section you'll create a Terraform module that can create an AWS Lambda and AWS IAM role. This module will include three files - `main.tf` which will contain the resource definitions, `variables.tf`
```

Comment:
Incomplete sentence or am I looking at the diff wrong?

Reply:
woops! Knew I left one dangling, good catch.